src/debug/websocket/ws-listener.js

```diff
@@ -10,13 +10,13 @@ BigInt.prototype.toJSON = function () {
 }
 
```

Comment:
should we add a prodUrl in here too?

Reply:
I haven't added it since we don't have a prod endpoint to test against. Ideally, we're not testing against the production endpoint anyway. I can add if we really want but seems unneeded to me at the moment